tests/test_definitions/column_map_expectations/expect_column_values_to_be_in_set.json

```diff
@@ -240,6 +243,22 @@
         "unexpected_index_list": [0],
         "unexpected_list": ["2018-01-01T00:00:00"]
       }
-    }]
+    },
+    {
+      "title": "basic_negative_test_case_datetime_set_bigquery",
+      "only_for": ["bigquery"],
```

Comment:
`expect_column_values_to_be_in_set` required a slightly different test for comparing `datetime` in bigquery